core/services/ocr3/plugins/ccip_integration_tests/helpers.go

```diff
@@ -259,15 +267,149 @@ func setupHomeChain(t *testing.T, owner *bind.TransactOpts, backend *backends.Si
 	require.NoError(t, err, "failed to add capabilities to the capability registry")
 	backend.Commit()
 
+	// Add NodeOperator, for simplicity we'll add one NodeOperator only
+	// First NodeOperator will have NodeOperatorId = 1
+	_, err = capabilityRegistry.AddNodeOperators(owner, []kcr.CapabilitiesRegistryNodeOperator{
+		{
+			Admin: owner.From,
+			Name:  "NodeOperator",
+		},
+	})
+	require.NoError(t, err, "failed to add node operator to the capability registry")
+	backend.Commit()
+
 	return homeChain{
 		backend:            backend,
 		owner:              owner,
 		chainID:            homeChainID,
 		capabilityRegistry: capabilityRegistry,
-		ccipConfigContract: capabilityConfig.Address(),
+		ccipConfig:         capabilityConfig,
 	}
 }
 
+func sortP2PIDS(p2pIDs [][32]byte) {
+	sort.Slice(p2pIDs, func(i, j int) bool {
+		return hex.EncodeToString(p2pIDs[i][:]) < hex.EncodeToString(p2pIDs[j][:])
```

Comment:
nit: no need to encode the bytes:
```suggestion
		return bytes.Compare(p2pIDs[i][:], p2pIDs[j][:])
```